### DISCUSSION TITLE
Support table column header actions when there is no panel title

### DISCUSSION THREAD

**Discussion by philka2** [grafana/grafana#38388] [Dashboards, Panels & Visualizations]
**What happened**:
Tables without a panel title are not sortable anymore. The tables "option menu header pop up" avoids to be able to click on the columns attributes name.

**What you expected to happen**:
Tables without a panel title are sortable by clicking on the columns attribute name.

**How to reproduce it (as minimally and precisely as possible)**:
![table_not_sortable_without_title](https://user-images.githubusercontent.com/6825178/84324259-ad4ab100-ab78-11ea-9617-79e4cbe51bdc.gif)

**Environment**:
- Grafana version: v7.0.3
- Data source type & version: Postgres 10
- OS Grafana is installed on: AWS ECS
- User OS & Browser: MacOs Chrome 83.0.4103.97
- Grafana plugins: -
- Others: -

**Comment by torkelo** [comment]
>  The tables "option menu header pop up"  

The what? 

I do not think this use case ever worked. We would need to update how the no panel title menu action works, so this is not easy to fix

**Comment by philka2** [comment]
> The what?

😃 

The table column header actions worked with no panel title for the old tables. When migrating my dashboards from v5 to v7 and switching to the new the table visualization the sort functionality was gone, so I just was wondering. It is not a big deal, you can also add a single space title as workaround if you really wish not having table titles.

**Comment by jcmcken** [comment]
Also have this issue. It used to be that it was easily sortable. At some later version, the editor menu would overshadow most of the column if you tried to click it, but you could still squeeze a click in with good aim. Now the editor menu covers everything and it's impossible to sort without adding a title to the table.

In our use case, we have dashboards that have a single data table with thousands of paginated data rows that's used for exploratory purposes. It's pointless to have both a dashboard title and a table title, since there's only a single visualization for the whole dashboard, so we removed the table title. We ended up having to add it back to get the sort working

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
